SecurityManager/Core/Domain/AccessControl/AccessEvaluation/SecurityContextRevisionBasedCacheBase.cs

```diff
@@ -84,7 +84,8 @@ private IExecutableQuery<IEnumerable<T>> CreateExecutableQuery<T> (string key, F
             "<dynamic query>",
             queryExecutor.StorageProviderDefinition,
             queryModel,
-            Enumerable.Empty<FetchQueryModelBuilder>());
+            Enumerable.Empty<FetchQueryModelBuilder>(),
+            queryExecutor.Metadata);
```

Comment:
In https://re-motion.atlassian.net/browse/RM-9164 you mentioned every place the metadata is exposed. Please update to mention where the metadata is part of the public api only, not when it is used internally.